plugins/qlcplus.js

```diff
@@ -19,11 +19,14 @@ module.exports.export = function exportQLCplus(library, options) {
     for (const mode of fixture.modes) {
       for (let i=0; i<mode.channels.length; i++) {
         if (mode.channels[i] == null) {
-          mode.channels[i] = 'No Function ' + i;
```

Comment:
You can not reuse the same channel more than once in a mode! Does the new code account for that?